espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/settings/DriverSetting.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.model.settings
+
+import io.appium.espressoserver.EspressoServerRunnerTest
+import io.appium.espressoserver.lib.drivers.DriverContext
+import io.appium.espressoserver.lib.handlers.exceptions.InvalidArgumentException
+
+class DriverSetting : AbstractSetting() {
+    override var name: String = "driver"
+
+    override fun value(): String {
+        return EspressoServerRunnerTest.context.driverStrategy.name.toString().toLowerCase()
+    }
+
+    override fun apply(value: Any?) {
+        try {
+            val driverStrategy = DriverContext.StrategyType.valueOf(value.toString().toUpperCase())
+            EspressoServerRunnerTest.context.setDriverStrategy(driverStrategy)
+        }
+        catch(e: IllegalArgumentException){
```

Comment:
we usually put `catch` into the same line with the closing bracket

Reply:
changed